cmds/dummy-oauth/main.go

```diff
@@ -0,0 +1,76 @@
+// ?grant_type=client_credentials&scope={}&intended_audience={}
+package main
+
+import (
+  "crypto/rsa"
+  "encoding/json"
+  "flag"
+  "fmt"
+  "io/ioutil"
+  "log"
+  "net/http"
+
+  "github.com/dgrijalva/jwt-go"
+)
+
+var (
+  address    = flag.String("addr", ":8085", "address")
+  keyFile    = flag.String("private_key_file", "config/test-certs/oauth.key", "oauth private key file")
+  privateKey *rsa.PrivateKey
+)
+
+// TODO(steeling): add other parameters so we can test expired tokens, invalid tokens, etc.
+func getToken(w http.ResponseWriter, r *http.Request) {
+  fmt.Println(r)
```

Comment:
fmt.infof.

Reply:
infof? I don't see it here https://golang.org/pkg/fmt/?m=all